README.rst

```diff
@@ -1,7 +1,7 @@
 Amaranth Yosys distribution
 ###########################
 
-`Amaranth <https://github.com/amaranth/amaranth>`_ is a Python-based hardware description language that uses `Yosys <http://www.clifford.at/yosys>`_ as a backend to emit Verilog.
+`Amaranth <https://github.com/amaranth-lang/amaranth>`_ is a Python-based hardware description language that uses `Yosys <http://bygone.clairexen.net/yosys>`_ as a backend to emit Verilog.
```

Comment:
Correct link to Yosys is actually https://yosyshq.net/yosys/

Reply:
Fixed, thanks!